rules/windows/initial_access_evasion_suspicious_htm_file_creation.toml

```diff
@@ -2,7 +2,9 @@
 creation_date = "2022/07/03"
 integration = ["endpoint"]
 maturity = "production"
-updated_date = "2024/05/21"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
```

Comment:
Why are we minstacking to older versions? We maintain packagings till 8.10 and release only from 8.12